cyw43/src/bus.rs

```diff
@@ -50,42 +50,90 @@ where
 
     pub async fn init(&mut self) {
         // Reset
+        defmt::debug!("WL_REG off/on");
         self.pwr.set_low().unwrap();
         Timer::after(Duration::from_millis(20)).await;
         self.pwr.set_high().unwrap();
         Timer::after(Duration::from_millis(250)).await;
 
+        defmt::debug!("read REG_BUS_TEST_RO");
         while self
-            .read32_swapped(REG_BUS_TEST_RO)
+            .read32_swapped(FUNC_BUS, REG_BUS_TEST_RO)
```

Comment:
i did this for "clarity", i think it's a good improvement (change the signature of read32_swapped / write32_swapped)

i can revert if needed

Reply:
:+1: